code/Extension/UserFormFieldEditorExtension.php

```diff
@@ -103,7 +104,8 @@ public function getFieldEditorGrid()
                 new GridFieldDeleteAction(),
                 new GridFieldToolbarHeader(),
                 new GridFieldOrderableRows('Sort'),
-                new GridFieldDetailForm(null, false, false)
+                new GridFieldDetailForm(),
+                new GridFieldPaginator(999)
```

Comment:
I don't think this line is necessary, is there a reason you've included it?
If it is necessary in order to add the better buttons, perhaps a comment could indicate that.

Reply:
Hi NightJar, thanks for reviewing the code.
if you leave out the line of the GridfieldPaginator, the prev next buttons will not be visible in the detail form.
So I think it is necessary to determine the which record is next or prev.

Reply:
Yes, I see that on deeper inspection. It is unfortunate this is the case, however it is what it is.
Thanks for adding the comment, that will help remove confusion for future readers :)